### PR TITLE
Add docs for switch to device_id for LCN actions

### PR DESCRIPTION
**Pull request by alengwenus**
## Proposed change
Documentation for the parameter change from ´address` to `device_id` for LCN actions.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/129590
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

## Summary by CodeRabbit

- **New Features**
	- Introduced a new action `address_to_device_id` for obtaining `device_id` from LCN modules or group addresses.
  
- **Documentation**
	- Updated action signatures to replace `address` with `device_id` for multiple actions, enhancing device identification.
	- Minor textual adjustments made for improved clarity and readability.

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -134,10 +134,9 @@ GEM
     ruby2_keywords (0.0.5)
     safe_yaml (1.0.5)
     sass (3.4.25)
-    sass-embedded (1.80.4-arm64-darwin)
-      google-protobuf (~> 4.28)
-    sass-embedded (1.80.4-x86_64-linux-gnu)
+    sass-embedded (1.80.4)
```
Comment (c0ffeeca7) [inline Gemfile.lock]:
```suggestion
    sass-embedded (1.80.4-arm64-darwin)
      google-protobuf (~> 4.28)
    sass-embedded (1.80.4-x86_64-linux-gnu)
```

File: Gemfile.lock [hunk 2]
```diff
@@ -134,10 +134,9 @@ GEM
...
       google-protobuf (~> 4.28)
+      rake (>= 13)
```
Comment (c0ffeeca7) [inline Gemfile.lock]:
```suggestion
```

File: source/_integrations/lcn.markdown [hunk 3]
```diff
@@ -431,13 +431,67 @@ For an explanation of the attributes refer to the corresponding [events](#events
...
+
+LCN modules and groups are identified by a combination of a numeric ID and a segment ID.
+
+The module/group ID typically ranges from 5 to 254. Segments can be addressed by their numeric ID (5 to 128) or 0 (indicating no segment).
+
+If multiple LCN integrations are set up, the specific integration is identified by its host's `name` (integration title) set in the integration panel.
+
+| Data attribute         | Optional | Description                       | Values                |
+| ---------------------- | -------- | --------------------------------- | --------------------- |
+| `id`                   | No       | Module or group id                | 5-254                |
```
Comment (c0ffeeca7) [inline source/_integrations/lcn.markdown]:
```suggestion
| `id`                   | No       | Module or group ID                | 5-254                |
```

File: source/_integrations/lcn.markdown [hunk 4]
```diff
@@ -431,13 +431,67 @@ For an explanation of the attributes refer to the corresponding [events](#events
...
+| `segment_id`           | Yes      | Segment id (default: 0)           | (0, 5-128)           |
```
Comment (c0ffeeca7) [inline source/_integrations/lcn.markdown]:
```suggestion
| `segment_id`           | Yes      | Segment ID (default: 0)           | (0, 5-128)           |
```

File: source/_integrations/lcn.markdown [hunk 5]
```diff
@@ -457,9 +511,9 @@ data:
 
 Set relative brightness of output port in percent.
 
-| Data attribute | Optional | Description                       | Values                |
+| Data attribute         | Optional | Description                       | Values                |
 | ---------------------- | -------- | --------------------------------- | --------------------- |
-| `address`              | No       | [LCN address](#lcn-addresses)     |
+| `device_id`            | No       | Home Assistant device id          ||
```
Comment (c0ffeeca7) [inline source/_integrations/lcn.markdown:464]:
```suggestion
| `device_id`            | No       | Home Assistant device ID          ||
```

File: source/_integrations/lcn.markdown [hunk 6]
```diff
@@ -478,9 +532,9 @@ data:
 
 Toggle output port.
 
-| Data attribute | Optional | Description                       | Values                |
+| Data attribute         | Optional | Description                       | Values                |
 | ---------------------- | -------- | --------------------------------- | --------------------- |
-| `address`              | No       | [LCN address](#lcn-addresses)     |
+| `device_id`            | No       | Home Assistant device id          ||
```
Comment (c0ffeeca7) [inline source/_integrations/lcn.markdown:485]:
```suggestion
| `device_id`            | No       | Home Assistant device ID          ||
```

File: source/_integrations/lcn.markdown [hunk 7]
```diff
@@ -501,35 +555,35 @@ Each character represents the state change of a relay (1=on, 0=off, t=toggle, -=
 
 Example states:  `t---001-`
 
-| Data attribute | Optional | Description                   | Values |
+| Data attribute         | Optional | Description                   | Values |
 | ---------------------- | -------- | ----------------------------- | ------ |
-| `address`              | No       | [LCN address](#lcn-addresses) |
-| `state`                | No       | Relay states as string        |
+| `device_id`            | No       | Home Assistant device id      ||
```
Comment (c0ffeeca7) [inline source/_integrations/lcn.markdown:508]:
```suggestion
| `device_id`            | No       | Home Assistant device ID      ||
```

File: source/_integrations/lcn.markdown [hunk 8]
```diff
@@ -501,35 +555,35 @@ Each character represents the state change of a relay (1=on, 0=off, t=toggle, -=
...
 
 ### Action: `led`
 
 Set the LED status.
 
-| Data attribute | Optional | Description                   | Values               |
+| Data attribute         | Optional | Description                   | Values               |
 | ---------------------- | -------- | ----------------------------- | -------------------- |
-| `address`              | No       | [LCN address](#lcn-addresses) |
+| `device_id`            | No       | Home Assistant device id      ||
```
Comment (c0ffeeca7) [inline source/_integrations/lcn.markdown:526]:
```suggestion
| `device_id`            | No       | Home Assistant device ID      ||
```

File: source/_integrations/lcn.markdown [hunk 9]
```diff
@@ -540,9 +594,9 @@ Set the absolute value of a variable or setpoint.
 If `value` is not defined, it is assumed to be 0.
 If `unit_of_measurement` is not defined, it is assumed to be `native`.
 
-| Data attribute | Optional | Description                   | Values                                                             |
+| Data attribute         | Optional | Description                   | Values                                                             |
 | ---------------------- | -------- | ----------------------------- | ------------------------------------------------------------------ |
-| `address`              | No       | [LCN address](#lcn-addresses) |
+| `device_id`            | No       | Home Assistant device id      ||
```
Comment (c0ffeeca7) [inline source/_integrations/lcn.markdown:547]:
```suggestion
| `device_id`            | No       | Home Assistant device ID      ||
```

File: source/_integrations/lcn.markdown [hunk 10]
```diff
@@ -569,9 +623,9 @@ Set the relative value of a variable or setpoint.
 If `value` is not defined, it is assumed to be 0.
 If `unit_of_measurement` is not defined, it is assumed to be `native`.
 
-| Data attribute | Optional | Description                   | Values                                                                                                |
+| Data attribute         | Optional | Description                   | Values                                                                                                |
 | ---------------------- | -------- | ----------------------------- | ----------------------------------------------------------------------------------------------------- |
-| `address`              | No       | [LCN address](#lcn-addresses) |
+| `device_id`            | No       | Home Assistant device id      ||
```
Comment (c0ffeeca7) [inline source/_integrations/lcn.markdown:576]:
```suggestion
| `device_id`            | No       | Home Assistant device ID      ||
```

File: source/_integrations/lcn.markdown [hunk 11]
```diff
@@ -596,17 +650,17 @@ Otherwise the module might show unexpected behavior or return error messages.
 
 Reset value of variable or setpoint.
 
-| Data attribute | Optional | Description                   | Values                                                             |
+| Data attribute         | Optional | Description                   | Values                                                             |
 | ---------------------- | -------- | ----------------------------- | ------------------------------------------------------------------ |
-| `address`              | No       | [LCN address](#lcn-addresses) |
+| `device_id`            | No       | Home Assistant device id          ||
```
Comment (c0ffeeca7) [inline source/_integrations/lcn.markdown:603]:
```suggestion
| `device_id`            | No       | Home Assistant device ID          ||
```

File: source/_integrations/lcn.markdown [hunk 12]
```diff
@@ -620,9 +674,9 @@ Otherwise the module might show unexpected behavior or return error messages.
 Locks a regulator setpoint.
 If `state` is not defined, it is assumed to be `False`.
 
-| Data attribute | Optional | Description                   | Values                           |
+| Data attribute         | Optional | Description                   | Values                           |
 | ---------------------- | -------- | ----------------------------- | -------------------------------- |
-| `address`              | No       | [LCN address](#lcn-addresses) |
+| `device_id`            | No       | Home Assistant device id      ||
```
Comment (c0ffeeca7) [inline source/_integrations/lcn.markdown:627]:
```suggestion
| `device_id`            | No       | Home Assistant device ID      ||
```

File: source/_integrations/lcn.markdown [hunk 13]
```diff
@@ -644,9 +698,9 @@ If `state` is not defined, it is assumed to be `hit`.
 The command allows the sending of keys immediately or deferred. For a deferred sending the attributes `time` and `time_unit` have to be specified. For deferred sending, the only key state allowed is `hit`.
 If `time_unit` is not defined, it is assumed to be `seconds`.
 
-| Data attribute | Optional | Description                   | Values                            |
+| Data attribute         | Optional | Description                   | Values                            |
 | ---------------------- | -------- | ----------------------------- | --------------------------------- |
-| `address`              | No       | [LCN address](#lcn-addresses) |
+| `device_id`            | No       | Home Assistant device id      ||
```
Comment (c0ffeeca7) [inline source/_integrations/lcn.markdown:651]:
```suggestion
| `device_id`            | No       | Home Assistant device ID      ||
```

File: source/_integrations/lcn.markdown [hunk 14]
```diff
@@ -681,10 +735,10 @@ The key lock states are defined as a string with eight characters. Each characte
 The command allows the locking of keys for a specified time period. For a time period, the attributes `time` and `time_unit` have to be specified. For a time period, only table `a` is allowed.
 If `time_unit` is not defined, it is assumed to be `seconds`.
 
-| Data attribute | Optional | Description                   | Values                            |
+| Data attribute         | Optional | Description                   | Values                            |
 | ---------------------- | -------- | ----------------------------- | --------------------------------- |
-| `address`              | No       | [LCN address](#lcn-addresses) |
-| `table`                | Yes      | Table with keys to lock       |
+| `device_id`            | No       | Home Assistant device id      ||
```
Comment (c0ffeeca7) [inline source/_integrations/lcn.markdown:688]:
```suggestion
| `device_id`            | No       | Home Assistant device ID      ||
```

File: source/_integrations/lcn.markdown [hunk 15]
```diff
@@ -716,19 +770,18 @@ Send dynamic text to LCN-GTxD displays.
...
 Each row can be set independently and can store up to 60 characters (encoded in UTF-8).
 
-
-| Data attribute | Optional | Description                        | Values |
+| Data attribute         | Optional | Description                        | Values |
 | ---------------------- | -------- | ---------------------------------- | ------ |
-| `address`              | No       | [LCN address](#lcn-addresses)      |
-| `row`                  | No       | Text row 1..4                      |
-| `text`                 | No       | Text to send for the specified row |
+| `device_id`            | No       | Home Assistant device id           ||
```
Comment (c0ffeeca7) [inline source/_integrations/lcn.markdown:723]:
```suggestion
| `device_id`            | No       | Home Assistant device ID           ||
```

File: source/_integrations/lcn.markdown [hunk 16]
```diff
@@ -737,17 +790,17 @@ data:
 
 Send arbitrary PCK command. Only the command part of the PCK command has to be specified in the `pck` string.
 
-| Data attribute | Optional | Description                   | Values |
+| Data attribute         | Optional | Description                   | Values |
 | ---------------------- | -------- | ----------------------------- | ------ |
-| `address`              | No       | [LCN address](#lcn-addresses) |
-| `pck`                  | No       | PCK command                   |
+| `device_id`            | No       | Home Assistant device id      ||
```
Comment (c0ffeeca7) [inline source/_integrations/lcn.markdown:743]:
```suggestion
| `device_id`            | No       | Home Assistant device ID      ||
```

**Review by c0ffeeca7 — DISMISSED** [review]
Thank you, @alengwenus :+1:

**Review by silamon — APPROVED** [review]

**Pull request merged**